### PR TITLE
Alumni page

### PR DESCRIPTION
**Pull request by Pratik2026**
This PR adds an alumni page showcasing past year's coordinator and co-coordinators of TPC.

**Screencapture:**

[Screencast from 07-12-23 09:29:43 AM IST.webm](https://github.com/BitByte-TPC/TPC-website/assets/128511266/936591fb-304d-47a1-bead-aa705d0fa699)

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: frontend/src/components/Cards/AlumniCard.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      width: "350px",
+      background: "transparent",
+      border: "2px solid cyan",
+      borderRadius: "24px",
+      boxShadow: "0px 0px 14px 5px #11BFD7",
+    },
```
Comment (Aksh-Bansal-dev) [inline frontend/src/components/Cards/AlumniCard.tsx]: Avoid hard-coding values whenever possible. You can use [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) instead.

**Review by Aksh-Bansal-dev — APPROVED** [review]
LGTM

**Comment by garg3133** [comment]
A few suggestions:
* Since we are on the Alumni Page, it is already implied that all these people are ex co/coco, no need to explicitly mention that. Just keep "Coordinator (2016-17)" (for example).
* Some people have been Co as well as Coco (like Anuraag sir), so you should mention that as well.
* Arrange the list in some order (~it is random right now~), either the most recent position holders first or the earliest first. (My bad, I got confused by the mix-up of batches in the list, but if you place Co before Coco in the list for a particular tenure, then the people from the same batch will be placed together in the list for the current earliest first arrangement.)
* Add Aksh and Himanshu to the list as well, as "Co-coordinator (2021-22)".

File: frontend/src/pages/Alumni/alumniList.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  {
+    name: "Manan Jethwani",
+    description: "Ex-Co-Coordinator(2020-22)",
+    profileImg: "Manan.jpeg",
+    linkedInProfileUrl: "https://www.linkedin.com/in/manan-jethwani",
+    githubProfileUrl: "https://www.github.com/mananjethwani",
+    email: "",
+  },
+  {
+    name: "Tara Sinh Aatrey",
```
Comment (garg3133) [inline frontend/src/pages/Alumni/alumniList.ts:75]: ~Tara~ Taara

**Pull request merged**